vue/src/Version/VersionEdit.vue

```diff
@@ -85,31 +85,31 @@
             <h3>{{ translate('TagManager_ChangesSinceLastVersion') }}:</h3>
             <table v-content-table>
               <thead>
-                <tr>
```

Comment:
It's definitely not a deal breaker, but shouldn't these lines be indented within the parent element? Same with the lines below?

Reply:
Removed the unwanted formatting